aten/src/ATen/native/BackendSelectRegister.cpp

```diff
@@ -0,0 +1,615 @@
+  return op.callUnboxedWithDispatchKey<Tensor, Scalar, Scalar, const TensorOptions &>(key, start, end, options);
+}
+
+Tensor arange_start_step(Scalar start, Scalar end, Scalar step, const TensorOptions & options) {
+  DispatchKey key = options.computeDispatchKey();
+  static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::arange", "start_step");
+  return op.callUnboxedWithDispatchKey<Tensor, Scalar, Scalar, Scalar, const TensorOptions &>(key, start, end, step, options);
+}
+
+Tensor bartlett_window(int64_t window_length, const TensorOptions & options) {
+  DispatchKey key = options.computeDispatchKey();
+  static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::bartlett_window", "");
+  return op.callUnboxedWithDispatchKey<Tensor, int64_t, const TensorOptions &>(key, window_length, options);
+}
+
+Tensor bartlett_window_periodic(int64_t window_length, bool periodic, const TensorOptions & options) {
+  DispatchKey key = options.computeDispatchKey();
+  static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::bartlett_window", "periodic");
+  return op.callUnboxedWithDispatchKey<Tensor, int64_t, bool, const TensorOptions &>(key, window_length, periodic, options);
+}
+
+Tensor blackman_window(int64_t window_length, const TensorOptions & options) {
+  DispatchKey key = options.computeDispatchKey();
+  static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::blackman_window", "");
+  return op.callUnboxedWithDispatchKey<Tensor, int64_t, const TensorOptions &>(key, window_length, options);
+}
+
+Tensor blackman_window_periodic(int64_t window_length, bool periodic, const TensorOptions & options) {
+  DispatchKey key = options.computeDispatchKey();
+  static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::blackman_window", "periodic");
+  return op.callUnboxedWithDispatchKey<Tensor, int64_t, bool, const TensorOptions &>(key, window_length, periodic, options);
+}
+
+Tensor empty(IntArrayRef size, const TensorOptions & options, c10::optional<MemoryFormat> memory_format) {
+  DispatchKey key = options.computeDispatchKey();
+  static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::empty", "memory_format");
+  return op.callUnboxedWithDispatchKey<Tensor, IntArrayRef, const TensorOptions &, c10::optional<MemoryFormat>>(key, size, options, memory_format);
+}
+
+Tensor empty_strided(IntArrayRef size, IntArrayRef stride, const TensorOptions & options) {
+  DispatchKey key = options.computeDispatchKey();
+  static auto op = c10::Dispatcher::singleton().findSchemaOrThrow("aten::empty_strided", "");
+  return op.callUnboxedWithDispatchKey<Tensor, IntArrayRef, IntArrayRef, const TensorOptions &>(key, size, stride, options);
+}
+
+Tensor empty_affine_quantized(IntArrayRef size, const TensorOptions & options, double scale, int64_t zero_point, c10::optional<MemoryFormat> memory_format) {
```

Comment:
shouldn't this be _empty_affine_quantized?

Reply:
renamed